CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+### Added
+- Rundeck "prepare-continous integration" rundeck job can now be used to upgrade an existing git repository (#110)
+
+### Changed
+- Bugfix - invalid settings.xml generated on maven slave when HTTP Proxy env var set (#106)
```

Comment:
Those should be under `Fixed` then :)

I'm going to update all changelogs today when I do the release - so I'm going to replace this with a slightly adapted version.